libs/positions/src/lib/positions-table.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import { render } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+import { MockedProvider } from '@apollo/react-testing';
+import PositionsTable from './positions-table';
+
+describe('PositionsTable', () => {
```

Comment:
Would be good to add a test to render a single row and assert that the right columns are shown and they are correctly formatted, I've done something similar already in order-list.spec.tsx